### DISCUSSION TITLE
立秋那天的夏夜晚风 | /posts/summer-autumn/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#584] [Announcements]
晚风轻拂澎湖湾，白浪逐沙滩，没有椰林缀斜阳，只是一片海蓝蓝。–《外婆的澎湖湾》

https://yuanfan.rbind.io/posts/summer-autumn/

**Comment by liechi** [comment]
照片很好看。

**Reply by earfanfan** [reply]
哈，人家蓝溪镇都断断续续耕了好多篇了，你撒时候不再搁笔耕几篇呢？那篇用迅哥儿视角写毛泽东思想的是写着写着写没了嘛

**Reply by liechi** [reply]
哈哈哈，各更各的，不攀比。

那篇先不写了。等想得成熟些再写。

**Comment by hongtaoh** [comment]
图三确实很真好。了凡居士摄影很有天赋。
